src/plugins/intel_gpu/src/graph/deconvolution.cpp

```diff
@@ -154,12 +154,12 @@ std::vector<layout> deconvolution_inst::calc_output_layouts(deconvolution_node c
                                        0,
                                        "User-defined size of output layout must be positive (>= 1)");
 
-        tensor output_size(input_layout.batch(),
-                           number_of_features,
-                           desc->output_size.spatial[0],
-                           desc->output_size.spatial[1],
-                           desc->output_size.spatial[2]);
-        return {layout{output_type, out_fmt, output_size}};
+        ov::PartialShape output_shape({input_layout.batch(),
```

Comment:
Use `ShapeType` please, also no need to change old behavior in calc_output_layout()